plugins/MIDIPressureToCC/PluginMIDIPressureToCC.hpp

```diff
@@ -0,0 +1,162 @@
+/*
+ * MIDI PressureToCC plugin based on DISTRHO Plugin Framework (DPF)
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * Copyright (C) 2019 Christopher Arndt <info@chrisarndt.de>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to
+ * deal in the Software without restriction, including without limitation the
+ * rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+ * sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+ * IN THE SOFTWARE.
+ */
+
+#ifndef PLUGIN_MIDIPressureTOCC_H
+#define PLUGIN_MIDIPressureTOCC_H
+
+#include "DistrhoPlugin.hpp"
+
+START_NAMESPACE_DISTRHO
+
+#ifndef MIN
+#define MIN(a,b) ( (a) < (b) ? (a) : (b) )
+#endif
+
+#ifndef MAX
+#define MAX(a,b) ( (a) > (b) ? (a) : (b) )
+#endif
+
+#ifndef CLAMP
+#define CLAMP(v, min, max) (MIN((max), MAX((min), (v))))
+#endif
+
+#define MIDI_CONTROL_CHANGE 0xB0
+#define MIDI_CHANNEL_PRESSURE 0xD0
+
+// -----------------------------------------------------------------------
+
+class PluginMIDIPressureToCC : public Plugin {
+public:
+    enum Parameters {
+        paramFilterChannel,
+        paramKeepOriginal,
+        paramCC,
+        paramCount
+    };
+
+    PluginMIDIPressureToCC();
+
+protected:
+    // -------------------------------------------------------------------
+    // Information
+
+    const char* getLabel() const noexcept override {
+        return "MIDIPressureToCC";
+    }
+
+    const char* getDescription() const override {
+        return "Convert Monophonic Channel Pressure (Aftertouch) into Control Change messages";
+    }
+
+    const char* getMaker() const noexcept override {
+        return "chrisarndt.de";
+    }
+
+    const char* getHomePage() const override {
+        return "https://chrisarndt.de/plugins/midipressuretocc";
+    }
+
+    const char* getLicense() const noexcept override {
+        return "https://spdx.org/licenses/MIT";
+    }
+
+    uint32_t getVersion() const noexcept override {
+        return d_version(1, 2, 2);
```

Comment:
Please start with version 1.0.0, I'll up the version to one indicating stable one, when I make a release.